core/includes/diff.inc

```diff
@@ -1080,6 +1220,9 @@ abstract class DiffFormatter {
     $this->endBlock();
   }
 
+  /**
+   * @return void
```

Comment:
As this is just a wrapper around ob_start(), we inherit that function's return, I guess.
https://www.php.net/manual/en/function.ob-start.php
Or is it null then?

Reply:
It's not actually returning anything, so it would be `void`, even though `ob_start()` returns `bool`. I wouldn't normally add `@return void` (though it does make it clearer) but needed a method comment.